Dockerfile.production

```diff
@@ -58,7 +58,9 @@ RUN bash -c "export DEBIAN_FRONTEND=noninteractive && \
 RUN bash -c "export DEBIAN_FRONTEND=noninteractive && \
     apt-get update -qq && \
     apt-get upgrade -y && \
-    apt-get install -y imagemagick libmagickwand-dev ghostscript poppler-utils xvfb"
+    apt-get install -y imagemagick libmagickwand-dev ghostscript poppler-utils xvfb && \
```

Comment:
What do we use ghostscript for? Image thumbnails? Wonder if we can switch to libvips which will reduce the layer size quite a bit - something for another PR though.

Reply:
I'm not completely sure — seems maybe to be related to file uploads, possibly for filetype detection?

I also wonder if there's any real benefit to having so many separate calls to apt-get.